xchange-core/src/main/java/org/knowm/xchange/dto/account/FundingRecord.java

```diff
@@ -304,7 +304,13 @@ public String toString() {
     INTERNAL_WITHDRAWAL(false),
 
     /** Used for transfers between exchanges accounts */
-    INTERNAL_DEPOSIT(true);
+    INTERNAL_DEPOSIT(true),
+
+    /** Used for realised losses from derivatives */
+    REALISED_LOSS(false),
+
+    /** Used for realised profits from derivatives */
+    REALISED_PROFIT(true);
```

Comment:
is this something that exists on many exchanges with derivatives or is it something specific to bitmex?

could you provide an example of another exchange that has a similar type in its API?

Reply:
Realised Profit and Loss, Realised PNL or RPL, is a standard term in trading derivatives. (There is also UnrealisedPNL but I think that can be represented through to status `PROCESSING`)

I know that at least Deribit and CMC Markets (not support jet) allow trading with derivatives. Deribits explicitly calls those earning RPL in their UI. Through their API they return a type called `settlement` with has a field `profit_loss`:
```
{
"type":"settlement"
"timestamp":1587110300141
"session_profit_loss":4469.243150033
"profit_loss":14.925158857
"position":78675
"mark_price":169.98
"instrument_name":"ETH-PERPETUAL"
"index_price":170.06
"funding":-0.008446859
}
```

Reply:
ok, thank you to explaining